syntax.php

```diff
@@ -72,11 +72,11 @@ function _link($target) {
         static $url = '^https?://';
         // '^(http://)?[-_[:alnum:]]+[-_.[:alnum:]]*\.[a-z]{2}'
         // '(/[-_./[:alnum:]&%?=#]*)?';
-        if (ereg($url, $target))
+        if (preg_match($url, $target))
```

Comment:
preg and ereg have slightly different syntax. the expression needs adjustment

Reply:
Works when delimiting the static:

`        static $url = '#^https?://#';`